### PR TITLE
Add suffixes to typos from "sp" to "st"

### PR DESCRIPTION
**Pull request by int-y1**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -49037,8 +49155,14 @@ streigtish->straightish
 streigtly->straightly
 streigtness->straightness
 streigts->straights, straits,
-strem->stream
+strem->stream, stem, strum,
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:49158]: `strep` (for `streptococcus`) although it is a specialised word?

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -49087,8 +49216,10 @@ strnad->strand
 strng->string
 strngs->strings
 stroage->storage
-stroe->store
-stroing->storing
+stroe->store, stroke, strobe, strove,
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:49219]: Also `strop` and `strode` (past of `stride`)?

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -48977,12 +49085,17 @@ stradegy->strategy
 strage->strange, storage,
 stragedy->strategy, tragedy,
 stragegy->strategy
+stragely->strangely
+strager->stranger
```
[the remark this answers is not in the document]
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:49089]: I couldn't find these typos.

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:49219]: I couldn't find `stroe->strop`. I'll add `strode`.

**Review by DimitriPapadopoulos — APPROVED** [review]

**Pull request merged**